pallets/staking-rewards/src/lib.rs

```diff
@@ -389,6 +453,8 @@ impl<T: Config> EraPayout<BalanceOf<T>> for Pallet<T> {
         total_issuance: BalanceOf<T>,
         era_duration_millis: u64,
     ) -> (BalanceOf<T>, BalanceOf<T>) {
+        pay_rent_rewards_out::<T>(pallet_staking::Pallet::<T>::active_era());
```

Comment:
Is there any other way to trigger this function call once per era? Like checking some condition in `on_initialize()` hook or something? And maybe not do it for all validators in one go for performance sake?
The `era_payout` is about inflation values calculation. Making over 50 transfers that are irrelevant to inflation seems not very appropriate here.

Reply:
we've discussed it already and posted a corresponding issue - https://github.com/gear-tech/gear/issues/3729